core/pom.xml

```diff
@@ -9,6 +10,71 @@
     <artifactId>smallrye-context-propagation</artifactId>
     <name>smallrye-context-propagation</name>
 
+    <properties>
+        <java9.sourceDirectory>${project.basedir}/src/main/java9</java9.sourceDirectory>
+        <java9.build.outputDirectory>${project.build.directory}/classes-java9</java9.build.outputDirectory>
+    </properties>
+
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>compile-java9</id>
```

Comment:
Shouldn't we now enforce min JDK version to build the project?

Reply:
Ah, good point.